test/e2e/specs/test.js

```diff
@@ -1,19 +1,77 @@
 // For authoring Nightwatch tests, see
 // http://nightwatchjs.org/guide#usage
+const d3 = require('d3');
+const _ = require('lodash');
+const failOnError = require('../helpers').failOnError;
 
 module.exports = {
-  'default e2e tests': function (browser) {
-    // automatically uses dev Server port from /config.index.js
-    // default: http://localhost:8080
-    // see nightwatch.conf.js
-    const devServer = browser.globals.devServerURL
+  'make space and new story': (browser) => {
+    const devServer = browser.globals.devServerURL;
 
-    browser
+    let yScale;
+    let xScale;
+
+    failOnError(browser)
       .url(devServer)
-      .waitForElementVisible('#app', 5000)
-      .assert.elementPresent('.hello')
-      .assert.containsText('h1', 'Welcome to Your Vue.js App')
-      .assert.elementCount('img', 1)
-      .end()
-  }
-}
+      .resizeWindow(1000, 800)
+      .waitForElementVisible('.modal .new-floorplan', 5000)
+      .setFlagOnError()
+      .click('.modal .new-floorplan')
+      .waitForElementVisible('#grid svg', 500)
+      .execute(
+        'return window.application.$store.state.application.scale', [],
+        ({ value: scale }) => {
+          xScale = d3.scaleLinear()
+            .domain(scale.x.rwuRange)
+            .range([0, scale.x.pixels]);
+          yScale = d3.scaleLinear()
+            .domain(scale.y.rwuRange)
+            .range([scale.y.pixels, 0]);
+        })
+      .perform((client, done) => {
+        console.log(`moving to ${xScale(-50)}, ${yScale(0)}`);
+        client
+        .waitForElementVisible('#grid svg', 200)
+        .pause(10)
+        .moveToElement('#grid svg', xScale(-50), yScale(0))
+        .waitForElementVisible('#grid svg .gridpoint', 200)
+        .pause(10)
+        .mouseButtonClick()
+        .pause(10)
+        .moveToElement('#grid svg', xScale(0), yScale(50))
+        .waitForElementVisible('.guideline-area-text', 100);
+
+        client.expect.element('.guideline-dist').text.to.contain('50');
+        client.expect.element('.guideline-area-text').text.to.contain('2,500');
```

Comment:
Might be worth double checking with NREL that we actually want to keep these numbers in the UI